src/Uno.UI.Toolkit/Uno.UI.Toolkit.Windows.csproj

```diff
@@ -65,8 +65,8 @@
 
 		<PropertyGroup>
 			<_OverrideTargetFramework>$(TargetFramework)</_OverrideTargetFramework>
-			<_OverrideTargetFramework Condition="'$(_OverrideTargetFramework)' == 'uap10.0.19041'">UAP</_OverrideTargetFramework>
-			<_TargetNugetFolder>$(NuGetPackageRoot)\Uno.WinUI\$(UnoNugetOverrideVersion)\lib\$(_OverrideTargetFramework)</_TargetNugetFolder>
+			<_OverrideTargetFramework Condition="'$(_OverrideTargetFramework)' == 'uap10.0.19041'">uap</_OverrideTargetFramework>
+			<_TargetNugetFolder>$(NuGetPackageRoot)\uno.winui\$(UnoNugetOverrideVersion)\lib\$(_OverrideTargetFramework)</_TargetNugetFolder>
```

Comment:
This doesn't do proper overriding in case of UWP tree. It is an existing issue though

Reply:
Right, we'll eventually need to move to the common override mecanism.